docs/index.md

```diff
@@ -0,0 +1,18 @@
+# Installation
```

Comment:
Use this style for
https://github.com/VirtoCommerce/vc-module-customer-review/blob/dev/docs/index.md